code/datums/components/twohanded.dm

```diff
@@ -266,6 +266,11 @@
 		return
 	if(held_item == parent)
 		return COMPONENT_BLOCK_SWAP
+	// check if we are holding an item with telekinesis, is that item our parent?
+	if(istype(held_item, /obj/item/tk_grab))
+		var/obj/item/tk_grab/tk_item = held_item
+		if (tk_item?.focus == parent)
```

Comment:
tk_item definitely exists, you istyped it on line 270
```suggestion
		if (tk_item.focus == parent)
```

Reply:
This was actually the first iteration of blocking TK from using items two handed.
It was replaced and i never removed this, this block is now gone.